filters/json.js

```diff
@@ -0,0 +1,62 @@
+// i18next j4 json format defined at https://www.i18next.com/misc/json-format
+import flatten from "flat";
+
+export class JsonFilter {
+    constructor(params) {
+        this.enableARBAnnotations = params?.enableARBAnnotations || false;
+        this.enablePluralSuffixes = params?.enablePluralSuffixes || false;
+        this.emitComments = params?.emitComments || false;
```

Comment:
the name is a bit misleading as JSON doesn't allow comments. maybe a better name would be `emitARBAnnotations`?

Reply:
Changed